pyensembl/compute_cache.py

```diff
@@ -8,7 +8,7 @@
 2) Cache warm on disk. Parse or unpickle the serialized result into memory.
 3) Cache warm in memory. Return cached object.
 """
-import cPickle
+import pickle
```

Comment:
Wow, good catch! Years ago, someone told me that pickle was aliased to cPickle starting in 2.7, an assertion that I never bothered to check, and is apparently wrong. I've been using pickle in so many places...

Fixed according to your suggestion.